mmv1/products/compute/TargetInstance.yaml

```
@@ -123,3 +135,10 @@ properties:
     default_value: :NO_NAT
     values:
       - :NO_NAT
+  - !ruby/object:Api::Type::String
+    name: 'securityPolicy'
+    min_version: beta
+    description: |
+      The resource URL for the security policy associated with this target instance.
+    immutable: true
```

Since this is handled by a separate endpoint, it seems like it should be possible to update in place without recreating the resource - does that sound right?

```suggestion
    update_url: projects/{{project}}/zones/{{zone}}/targetInstances/{{name}}/setSecurityPolicy
    update_verb: :POST
```

See https://googlecloudplatform.github.io/magic-modules/develop/resource/#add-fields for more information about update_url and update_verb for fields.

---

See https://googlecloudplatform.github.io/magic-modules/develop/resource/#add-fields - this field can be mutable even though the overall resource is not.